main/src/main/scala/sbt/Build.scala

```diff
@@ -8,6 +8,7 @@ import Keys.{ name, organization, thisProject, autoGeneratedProject }
 import Def.{ ScopedKey, Setting }
 
 // name is more like BuildDefinition, but that is too long
+@deprecated("Use .sbt format instead", "0.13.12")
```

Comment:
I think we should write a migration guide in http://www.scala-sbt.org/0.13/docs/Full-Def.html and link to it from here.